content/sv/dependencies.md

```diff
@@ -0,0 +1,12 @@
+## II. Dependencies (Beroenden)
+### Deklarera och isolera beroenden explicit
+
+De flesta programmeringsspråk erbjuder någon form av paketeringssystem för distribuering av stödbibliotek, som ex. [CPAN](http://www.cpan.org/) för Perl eller [Rubygems](http://rubygems.org/) för Ruby. Bibliotek installerade genom ett paketeringssystem kan installeras för hela arbetsmiljön (kallat för "systempaket", eng. "site packages") eller inom applikationens lokala utrymmet (kallat för "tredjepartspaket" eller "paketering", eng. "vendoring"/"bundling").
+
+**En tolvfaktorapplikation förlitar sig aldrig på en implicit tillgänglighet av systempaket.** Den deklararerar alla beroenden, komplett och exakt, via ett *manifest för beroende-deklaration* (eng. "dependency declaration manifest"). Vidare använder den verktyg för *isolering av beroenden* under exekvering för att försäkra sig om att inga implicita beroenden "läckt in" från det omgivande körsystemet. Den fullständiga och exakta beroendespecifikationen appliceras enhetligt på både produktionsmiljön och utvecklingsmiljön.
+
+Till exempel, [Bundler](https://bundler.io/) för Ruby erbjuder manifestformatet `Gemfile` för beroendedeklaration och `bundle exec` för beroendeisolering. I Python finns det två separata verktyg för dessa steg -- [Pip](http://www.pip-installer.org/en/latest/) används för deklaration och [Virtualenv](http://www.virtualenv.org/en/latest/) för isolering. Även C har [Autoconf](http://www.gnu.org/s/autoconf/) för beroendedeklaration, och statisk länkning kan användas som beroendeisolering. Oavsett vilka verktyg som används måste man använda beroendedeklaration och beroendeisolering tillsammans -- bara den ena eller den andra är inte tillräckligt för att tillfredsställa tolvfaktormodellen.
```

Comment:
```suggestion
Som exempel så används [Bundler](https://bundler.io/) för Ruby erbjuder manifestformatet `Gemfile` för beroendespecifikation och `bundle exec` för beroendeisolering. I Python finns det två separata verktyg för dessa steg -- [Pip](http://www.pip-installer.org/en/latest/) används för beroendespecifikation och [Virtualenv](http://www.virtualenv.org/en/latest/) för isolering. Även C har [Autoconf](http://www.gnu.org/s/autoconf/) för beroendespecifikation, och statisk länkning kan användas som beroendeisolering. Oavsett vilka verktyg som används måste man använda beroendespecifikation och beroendeisolering tillsammans -- bara den ena eller den andra är inte tillräckligt för att tillfredsställa tolvfaktormodellen.
```

Reply:
I agree with the change to "beroendespecifikation", however language needs to be adjusted somewhat. Am I able to make a countersuggestion somehow? Changes marked with bold and italics.

_**Exempelvis**_ används [Bundler](https://bundler.io/) för Ruby **_som_** erbjuder manifestformatet `Gemfile` för beroendespecifikation och `bundle exec` för beroendeisolering. I Python finns det två separata verktyg för dessa steg -- [Pip](http://www.pip-installer.org/en/latest/) **_som_** används för beroendespecifikation och [Virtualenv](http://www.virtualenv.org/en/latest/) **_som används_** för isolering. Även C har [Autoconf](http://www.gnu.org/s/autoconf/) för beroendespecifikation, och statisk länkning kan användas som beroendeisolering. Oavsett vilka verktyg som används måste man använda beroendespecifikation och beroendeisolering tillsammans -- bara den ena eller den andra är inte tillräckligt för att tillfredsställa tolvfaktormodellen.

Reply:
Excellent improvements 👍